packages/react-ui/src/components/molecules/Alert.d.ts

```diff
@@ -0,0 +1,11 @@
+import { AlertProps as MuiAlertProps } from '@mui/material/Alert';
+
+export type AlertProps = MuiAlertProps & {
```

Comment:
i am worried that we need to remove original `severity` from `MuiAlertProps` -> `Omit< MuiAlertProps, 'severity'>` ....

Have you checked it with typescript code, i'm confused when it's needed or not, so sorry if it's already ok